octobot_trading/exchanges/traders/trader.py

```diff
@@ -126,28 +125,135 @@ async def create_artificial_order(self, order_type, symbol, current_price, quant
                                                                     symbol=symbol,
                                                                     current_price=current_price,
                                                                     quantity=quantity,
-                                                                    price=price,
-                                                                    linked_portfolio=linked_portfolio))
-
-    async def _create_new_order(self, new_order: object, portfolio) -> object:
+                                                                    price=price))
+
+    async def edit_order(self, order: object,
+                         edited_quantity: decimal.Decimal = None,
+                         edited_price: decimal.Decimal = None,
+                         edited_stop_price: decimal.Decimal = None,
+                         edited_current_price: decimal.Decimal = None,
+                         params: dict = None) -> bool:
+        """
+        Edits an order, might be a simulated or a real order.
+        Fields that can be edited are:
+            quantity, price, stop_price and current_price
+        Portfolio is updated within this call
+        :return: True when an order field got updated
+        """
+        if not order.can_be_edited():
+            raise RuntimeError(f"Order can't be edited, order: {order}")
```

Comment:
Should we create a custom error here instead of raising a RuntimeError?

Reply:
Right